tests/nexus_aps_map.json

```diff
@@ -20,10 +20,10 @@
 	"/entry/sample/temperature": "/measurement/sample/temperature_A",
 	"/entry/sample/position_value": "/measurement/sample/translation",
 	"/entry/XPCS/instrument/mask/mask": "/xpcs/mask",
-	"/entry/XPCS/instrument/mask/dqmap": "/xpcs/dqmap",
-	"/entry/XPCS/instrument/mask/dqlist": "/xpcs/dqlist",
-	"/entry/XPCS/instrument/mask/dphilist": "/xpcs/dphilist",
-	"/entry/XPCS/instrument/mask/sqmap": "/xpcs/sqmap",
-	"/entry/XPCS/instrument/mask/sqlist": "/xpcs/sqlist",
-	"/entry/XPCS/instrument/mask/sphilist": "/xpcs/sphilist"
+	"/entry/XPCS/instrument/mask/dynamic_roi_map": "/xpcs/dqmap",
+	"/entry/XPCS/instrument/mask/dynamic_q_list": "/xpcs/dqlist",
+	"/entry/XPCS/instrument/mask/dynamic_phi_list": "/xpcs/dynamic_phi_list",
+	"/entry/XPCS/instrument/mask/static_roi_map": "/xpcs/sqmap",
+	"/entry/XPCS/instrument/mask/static_q_list": "/xpcs/static_q_list",
+	"/entry/XPCS/instrument/mask/static_phi_list": "/xpcs/sphilist"
 }
```

Comment:
I think this json file might be obsolete anyways, but to be consistent, the right side of the mapping would still need to be dphilist and sqlist as these come from the original APS files, where the path is named like this.